stories/plotly/plotly-bar.stories.js

```diff
@@ -1,26 +1,28 @@
 import React from 'react'
 import Bar from '../../src/components/plotly/bar'
-import { mockData } from '../data/others/mock-data'
-import ParentSize from '@visx/responsive/lib/components/ParentSize'
+
+import mockData from '../data/plotly/mock-data-pie-bar'
+import ResponsiveChartWrapper from '../nivo/responsive-chart-wrapper'
 
 export default {
   title: 'Plotly/bar',
 }
 
-const keyX = mockData.map(({ Name }) => Name)
-const keyY = mockData.map(({ Age }) => Age)
-
-const data = [
-  {
-    x: keyX,
-    y: keyY,
-    type: 'bar',
-  },
-]
-export const Default = () => <Bar data={data} />
+export const Default = () =>
```

Comment:
We are now using everywhere the new syntax for stories. Check out Do's `visx` charts and, of course, `https://storybook.js.org/docs/react/writing-stories/introduction`
The older stories will have to be re-written at some point. Not urgent at the moment.

Reply:
@geoerika I think I am using the new syntax in the plotly stories, no?

Reply:
Oh yeah, but there is a lot of repeated code here, you can make use of a template in your stories, something like this:
<img width="1395" alt="Screen Shot 2021-10-06 at 8 45 02 AM" src="https://user-images.githubusercontent.com/41120953/136205115-82d1f5f0-309a-43a2-aab6-a9dc1665dee1.png">

Reply:
Oh, right, will do. Thanks